### ISSUE TITLE
SideMenu extra white space on tap and pull from each side

### ISSUE THREAD

**Issue by cyclops24** [JoeyAndres/meteor-ionic#12]
Hi @JoeyAndres ,
I used Side Menu in all of my project one other issue that exist in meteoric version (and ok on ionic version) is when you tap and pull page to a side and pull more than side menu width an extra space occur between content and side menu and a small horizontal scroll shown below side menu. See below image for more info:
![screenshot from 2016-02-06 16 53 08](https://cloud.githubusercontent.com/assets/8266479/12866900/506e7614-ccf3-11e5-9660-2404e3a3cf76.png)

**Comment by cyclops24** [comment]
Hi again @JoeyAndres,
I think that I found a solution:
meteoric use [Snap.js](https://github.com/jakiestfu/Snap.js) and I see that it has a option named `hyperExtending`, I think set this option to false fix this issue (but I don't know where is the right place to set this).
See [this link](http://jakiestfu.github.io/Snap.js/demo/apps/hyperextend.html) for more detail please.

**Comment by JoeyAndres** [comment]
@cyclops24 Try it again. I havent changed ionic's hyperextending yet, but I removed PEP (Pointer Events Polyfill) and just used native events. PEP contains MutationObserver causing things to be SUPER SLOW and buggy.

Note: I introduced PEP since snap.js wasn't consistent with its native event handlers. I finally found a way to have a more robust native event handlers (thanks to reading iscroll's code) and this should now be fixed.
